Dockerfile

```diff
@@ -156,7 +158,14 @@ RUN --mount=from=containerd-src,src=/usr/src/containerd,readwrite --mount=target
   && make bin/containerd-shim \
   && mv bin /out
 
-FROM tonistiigi/registry:$REGISTRY_VERSION AS registry
+FROM gobuild-base AS registry
+ARG REGISTRY_VERSION
+RUN git clone https://github.com/docker/distribution.git /go/src/github.com/docker/distribution
```

Comment:
why do we need this. I fixed the Dockerfile for official registry image while back. I think we can just switch to that one now. https://github.com/docker/distribution/pull/2735

Reply:
fixed to use official image